src/PaylineProvider.tsx

```diff
@@ -1,10 +1,10 @@
-import React, { useLayoutEffect, useState } from 'react';
+import React, { PropsWithChildren, useLayoutEffect, useState } from 'react';
 
-type PropsType = {
+type PropsType = PropsWithChildren<{
   production?: boolean;
-};
+}>;
 
-export const PaylineHead: React.ComponentType<PropsType> = ({ production = false }) => {
+export const PaylineHead = ({ production = true }: PropsType) => {
```

Comment:
You turned PaylineHead production by default. It is intended to be staging first, for safety

Reply:
Oups 😬